README.md

```diff
@@ -10,12 +10,12 @@
 For fast start you can go through [tutorial](https://jdi-docs.github.io/jdi-light/?java#tutorial) </br>
 
 More links: </br>
-Documentation: <a href = "https://jdi-docs.github.io/jdi-light/?java" target = "a_blank"> https://jdi-docs.github.io/jdi-light/?java </a> - different types of documentation from introduction and tutorial to technical methods documentation </br>
-Introduction + fast start example: <a href = "https://jdi-docs.github.io/jdi-light/?java#introduction" target = "a_blank"> https://jdi-docs.github.io/jdi-light/?java#introduction </a> </br>
-Tutorial: <a href = "https://jdi-docs.github.io/jdi-light/?java#tutorial" target = "a_blank"> https://jdi-docs.github.io/jdi-light/?java#tutorial </a> - helps to get main ideas on practice </br>
-Tutorial repo: <a href = "https://github.com/jdi-tutorials" target = "a_blank"> https://github.com/jdi-tutorials </a> </br>
-Project templates: <a href = "https://github.com/jdi-templates" target = "a_blank"> https://github.com/jdi-templates </a>  - for fast projects start </br>
-Examples: <a href = "https://github.com/jdi-examples" target = "a_blank"> https://github.com/jdi-examples </a>  </br> 
+Documentation: https://jdi-docs.github.io/jdi-light/?java - different types of documentation from introduction and tutorial to technical methods documentation </br>
+Introduction + fast start example: https://jdi-docs.github.io/jdi-light/?java#introduction </br>
+Tutorial: https://jdi-docs.github.io/jdi-light/?java#tutorial - helps to get main ideas on practice </br>
+Tutorial repo:  https://github.com/jdi-tutorials </br>
+Project templates: https://github.com/jdi-templates - for fast projects start </br>
+Examples: https://github.com/jdi-examples </br>
 How to use [UI Elements examples](https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements) </br>
```

Comment:
It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169